source/docs/guide/src/ide_support.md

```diff
@@ -0,0 +1,146 @@
+# Adding IDE Support for Verus Using a Custom Version of rust-analyzer
+
+
+## Quickstart
+The steps below walk you through compiling a Verus-specific version of rust-analyzer and installing it in VS Code. After following the below steps, when you save a file, this custom rust-analyzer will run Verus for that file and display Verus results. It also provides Verus syntax support, IDE functionalities, and Verus-specific code actions.
+
+
+#### 1. Compile custom rust-analyzer
+
+1. Clone the repository: `git clone https://github.com/verus-lang/rust-analyzer/tree/november-08-2022`  
+2. `cd rust-analyzer`
+3. Compile the rust-analyzer binary: `cargo xtask dist`
+4. Unzip the generated file (e.g. `gunzip ./dist/rust-analyzer-x86_64-apple-darwin.gz`)
+5. Make it executable (e.g. `chmod +x ./dist/rust-analyzer-x86_64-apple-darwin`)
+
+
+
+#### 2. VS Code
+
+1. Please install the rust-analyzer extension in VS Code's extensions tab.
+2. We need to change the configuration inside `settings.json`. Specifically, we need to set the two variables below.
```

Comment:
Does this only affect the project's local configuration or is it global for VScode? (If it's the latter, you may want to warn the user.)